### PR TITLE
WIP: togglable sandbox syncer

### PR DESCRIPTION
**Pull request by shamsimam**
This PR builds on top of https://github.com/twosigma/Cook/pull/563 

It introduces a `sandbox-sync-enabled` flag to disable sandbox syncing from agent host updates. It also re-enables looking up the agent state to retrieve the `sandbox-directory` while loading an instance.

### CODE REVIEW

File: integration/tests/cook/test_basic.py [hunk 1]
```diff
@@ -32,16 +32,16 @@ def test_basic_submit(self):
         self.assertEqual(False, job['disable_mea_culpa_retries'])
         self.assertTrue(len(util.wait_for_output_url(self.cook_url, job_uuid)['output_url']) > 0)
 
-        job = util.wait_for_sandbox_directory(self.cook_url, job_uuid)
```
[the remark this answers is not in the document]
Comment (shamsimam) [reply integration/tests/cook/test_basic.py]: Done.

File: scheduler/src/cook/mesos/sandbox.clj [hunk 2]
```diff
@@ -159,46 +159,57 @@
...
-    (send task-id->sandbox-agent aggregate-sandbox task-id sandbox-directory)))
-
 (defn sync-agent-sandboxes
   "Asynchronously triggers state syncing from the mesos agent."
   [publisher-state framework-id agent-hostname]
   (when (and framework-id agent-hostname)
     (future
       (refresh-agent-cache-entry publisher-state framework-id agent-hostname))))
 
+(defn hostname->task-id->sandbox-directory
```
[the remark this answers is not in the document]
Comment (shamsimam) [reply scheduler/src/cook/mesos/sandbox.clj]: We need both the `hostname` and the `task-id`. I like leaving it in the name to indicate the two-level redirection required.

**Comment by shamsimam** [comment]
We've disabled the sandbox syncer. Closing the PR.

**Comment by dposada** [comment]
Can we delete the branch?

**Pull request closed without merging**